srcpkgs/frescobaldi/patches/0002-more-py-3.10-float-int-fixes-issue-1398.patch

```diff
@@ -0,0 +1,209 @@
+ 
+     def group(self):
+         """Return the group the page belongs to.
+-        
++
+         This could be some document structure, so that different Page objects
+         could refer to the same graphical contents, preventing double caching.
+-        
++
+         This object is used together with the value returned by ident() as a key
+         to cache the page. The idea is that the contents of the page are
+         uniquely identified by the objects returned by group() and ident().
+-        
++
+         This way, when the same document is opened in multiple page instances,
+         only one copy resides in the (global) cache.
+-        
++
+         By default, the page object itself is returned.
+-        
++
+         """
+         return self
+-        
++
+     def ident(self):
+         """Return a value that identifies the page within the group returned
+         by group().
+-        
++
+         By default, None is returned.
+-        
++
+         """
+         return None
+-    
++
+     def mapToPage(self, width=None, height=None):
+         """Return a MapToPage object, that can map original to Page coordinates.
+ 
+@@ -426,10 +426,10 @@ class AbstractPage(util.Rectangular):
+ 
+     def text(self, rect):
+         """Implement this method to get the text at the specified rectangle.
+-        
++
+         The rectangle should be in page coordinates. The default implementation
+         simply returns an empty string.
+-        
++
+         """
+         return ""
+ 
+@@ -437,22 +437,22 @@ class AbstractPage(util.Rectangular):
+         """Implement this method to load our links."""
+         from . import link
+         return link.Links()
+-    
++
+     def links(self):
+         """Return the Links object, containing Link objects.
+-        
++
+         Every Link denotes a clickable area on a Page, in coordinates 0.0-1.0.
+         The Links object makes it possible to quickly find a link on a Page.
+         This is cached after the first request, you should implement the
+         getLinks() method to load the links.
+-        
++
+         """
+         try:
+             return self._links
+         except AttributeError:
+             links = self._links = self.getLinks()
+         return links
+-    
++
+     def linksAt(self, point):
+         """Return a list() of zero or more links touched by QPoint point.
+ 
+@@ -468,9 +468,9 @@ class AbstractPage(util.Rectangular):
+ 
+     def linksIn(self, rect):
+         """Return an unordered set of links enclosed in rectangle.
+-        
++
+         The rectangle is in page coordinates.
+-        
++
+         """
+         return self.links().inside(*self.mapFromPage(1, 1).rect(rect).getCoords())
+ 
+@@ -481,9 +481,9 @@ class AbstractPage(util.Rectangular):
+ 
+ class AbstractRenderedPage(AbstractPage):
+     """A Page that has a renderer that performs caching and painting.
+-    
++
+     The renderer lives in the renderer attribute.
+-    
++
```

Comment:
Can you remove these whitespace changes from the patch as well?

Reply:
The changes that include whitespace in this patch are from a commit by the upstream maintainer themself. I remember discussing this on IRC and being advised to just leave it.